### PR TITLE
SetHideFlags now ORs HideFlags rather than setting. Changed defaults …

### PR DESCRIPTION
**Pull request by TimsLuxo**
…work better with default Usd Asset import settings. The old code overrode the Hide settings for instance Masters, making them visible in the Hierarchy scene graph.

### CODE REVIEW

File: package/com.unity.formats.usd/Samples/ImportProcessor/SetHideFlags.cs [hunk 1]
```diff
@@ -24,21 +24,38 @@ public class SetHideFlags : RegexImportProcessor, IImportPostProcessHierarchy {
 
     public HideFlags hideFlagsSettings = HideFlags.DontSave;
 
+    public enum Operation {Overwrite, LogicalAND, LogicalOR, LogicalXOR}
+    [Tooltip("The logical operator to use when setting the flags. In most cases Or will produce expectd results")]
```
Comment (jcowles) [inline package/com.unity.formats.usd/Samples/ImportProcessor/SetHideFlags.cs]: Typo in "expectd"

File: package/com.unity.formats.usd/Samples/ImportProcessor/SetHideFlags.cs [hunk 2]
```diff
@@ -24,21 +24,38 @@ public class SetHideFlags : RegexImportProcessor, IImportPostProcessHierarchy {
...
+          default:
+            go.hideFlags = hideFlagsSettings;
+            return;
+        }
       }
     }
 
     void Reset() {
-      matchExpression = "/";
+      matchExpression = "/World";
```
Comment (jcowles) [inline package/com.unity.formats.usd/Samples/ImportProcessor/SetHideFlags.cs]: Why "/World"?

**Comment by TimsLuxo** [comment]
If it’s set to / then donotsave is applied to the root, meaning the root
will disappear on save/reload.
Probably should be changed to use mode regex and “/.+”

**Comment by jcowles** [comment]
Sorry for the delay, LGTM!

**Pull request merged**